builtin/providers/aws/resource_aws_iam_policy.go

```diff
@@ -221,7 +248,8 @@ func readIamPolicy(d *schema.ResourceData, policy *iam.Policy) error {
 	if err := d.Set("arn", *policy.Arn); err != nil {
 		return err
 	}
-	// TODO: set policy
-
+	if err := d.Set("arn", *policy.Arn); err != nil {
```

Comment:
d.Set doesn't need to have a value dereferenced before being passed - it safely dereferences a pointer if it can :)